### DISCUSSION TITLE
Make discussions tab default and remove issues tab.

### DISCUSSION THREAD

**Discussion by Osiris-Team** [community/community#27804] [Discussions]
Discussions is basically a newer version of the issues tab, so I have no idea why it still exists.
At least remove it for new repos or make it opt-in.

Issues should also have the ability to get up or downvoted. Also the UI is much more pleasant on the discussions tab than the issues tab and allows for quicker navigation between topics and better organization of issues.

There is nothing that is special about the issues tab. Also **almost** everything that can be done there can be done better inside the discussions tab.

> Discussion threads cannot get a PR assigned to them, nor can't be assigned to specific users. They cannot be put on project boards or have 
> milestones connected to them. You cannot 'close' Discussions, and definitely not through commit messages.
>  by @mpboom

Those are valid points yeah. But it's stuff that can be fixed easily once discussions are made default.

**Comment by mpboom** [comment]
Well, they're not exactly the same. Discussions are more forum-like threads meant for, right, discussions on certain topics. Issues are nailed-down and specific things you can address, link a PR to and ship. I can definitely see both features co-exist and thrive within a single repository.

**Reply by Osiris-Team** [reply]
Nah, Issues should also have the ability to get up or downvoted. Also the UI is much more pleasant on the discussions tab than the issues tab and allows for quicker navigation between topics and better organization of issues.

**Comment by jfullstackdev** [comment]
you have that option at the repo settings to disable it but not by default because it will always depend on the need of that specific project

**Reply by Osiris-Team** [reply]
Yeah I know. Im saying we should switch that and make discussions tab the default and remove the old issues tab, or make it opt-in.

**Reply by jfullstackdev** [reply]
they are important still particularly for other projects, so we can't say it's the Discussion tab that should be the default, particularly for legacy, the update might break those repos that used extensively the Issues tab before Discussion feature was added 

well we don't know if they will be getting rid of Issues and that will be just one category of Discussion

**Reply by mpboom** [reply]
> the old issues tab

Note there is no "old issues tab", as Issues are complementary to Discussions.

**Reply by Osiris-Team** [reply]
I know that's what I don't get. Its the same as the issues tab, only better.

**Reply by Osiris-Team** [reply]
There is nothing that is special about the issues tab. Also everything that can be done there can be done better inside the discussions tab.

**Reply by jfullstackdev** [reply]
as I'm also interested with the progress of GitHub, maybe yes, and Issues will be just one of the categories under Discussions tab

**Reply by mpboom** [reply]
Discussion threads cannot get a PR assigned to them, nor can't be assigned to specific users. They cannot be put on project boards or have milestones connected to them. You cannot 'close' Discussions, and definitely not through commit messages.

**Reply by Osiris-Team** [reply]
Those are valid points yeah. But it's stuff that can be fixed easily once discussions are made default.
